config/prod/miac-data-model-web-ec2.yaml

```diff
@@ -0,0 +1,45 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2.yaml
```

Comment:
the example template has been updated. please use the latest one from here: https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/3/files

Reply:
Just updated to follow the updated template.